packages/flutter_tools/lib/src/commands/create.dart

```diff
@@ -41,7 +41,10 @@ class CreateCommand extends CreateBase {
       valueHelp: 'type',
       allowedHelp: <String, String>{
         flutterProjectTypeToString(FlutterProjectType.app): '(default) Generate a Flutter application.',
-        flutterProjectTypeToString(FlutterProjectType.package): 'Generate a shareable Flutter project containing only Dart code.',
+        flutterProjectTypeToString(FlutterProjectType.skeleton): 'Generate a List View / Detail View Flutter '
+            'application that follows community best practices',
```

Comment:
This should end with a period. If the tests didn't catch it, we should update the tests.

Reply:
Fixed.

Do you know which tests would catch this? I'm unable to run some of the tests locally, and due to the multi-package nature of this change, I can't see all the test results until 1) this CL is approved, 2) I land the `flutter/packages` change, 3) I re-run this tests on this change.